Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -1269,6 +1269,45 @@
   <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'OneDrive|x86'">
```

Comment:
Same as the others, OneDrive configuration should be removed?

Reply:
I'm not sure why this remained - I was trying to speed up my build by constraining it to just the OneDrive bits - I deleted the config so it shouldn't have been here - I'll stake it down and sprinkle it with some holy water this time.

I'll also removed the added references.